ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -100,6 +100,17 @@ - (void)reload
   }
 }
 
+- (void)reloadFromOrigin
```

Comment:
`reloadFromOrigin` does not seem to directly proxy the behavior from WKWebView. It might cause some confusion for people who try to utilize this. Is there any reason why you put in the special logic?